cuBERT_topic_modelling/tests/test_ctfidf.py

```diff
@@ -0,0 +1,105 @@
+from bertopic import BERTopic
+import pandas as pd
+import numpy as np
+import cudf
+from sklearn.datasets import fetch_20newsgroups
+import pytest
+from cupyx.scipy.sparse.csr import csr_matrix
+from cuBERTopic import gpu_BERTopic
+from ctfidf import ClassTFIDF
```

Comment:
Did you run `flake8/black` on the tests aswell ?

Reply:
Yep I did, will double check formatting :)